pkg/proxy/proxy.go

```diff
@@ -100,14 +99,32 @@ func (p *Proxy) Create(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	lto, err := p.kubeClient.CreateLoadTest(ctx, loadTest)
-	if err != nil {
-		if err == os.ErrExist {
+	// Find the old load test with same data
+	labeledLoadTests, err := p.kubeClient.ListLoadTests(ctx, loadTest)
+
+	if len(labeledLoadTests.Items) > 0 {
+
+		// If users wants to overwrite
+		if loadTest.Spec.Overwrite == true {
+			for _, item := range labeledLoadTests.Items {
+
+				// Remove the old tests
+				err := p.kubeClient.DeleteLoadTest(ctx, item.Name)
+				if err != nil {
+					logger.Error("Could not delete load test with error:", zap.Error(err))
+					render.Render(w, r, cHttp.ErrResponse(http.StatusBadRequest, err.Error()))
```

Comment:
Not sure that this is BadRequest from a user perspective. If we can delete load test should it be 500?